tests/installation/release_notes_from_rpm.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check origin of displayed release notes during installation
```

Comment:
You need to provide a meaningful commit message: "Fixes for poo#97316" References to ticket do not indicate what is about, for me...I don't care much, but other collaborator filter with git-log checking latest changes in the repo and they would like to see it in that way.

Please, update the summary of of both file, they are doing two difference things, so they should have different summary, one mentioning rpm the other mentioning url.